lib/vintage_net_wizard/backend_server.ex

```diff
@@ -278,6 +278,17 @@ defmodule VintageNetWizard.BackendServer do
     {:reply, state, state}
   end
 
+  def handle_call(
+        :complete,
+        _from,
+        %State{backend_state: %{data: %{configuration_status: :good}}} = state
```

Comment:
Yes, that does make sense. It fits with the existing pattern matching at that point.

I do have reservations. Partly it's because the default backend looks much harder to write a test for.

Mostly though, it's that the reason for not (re)applying the configuration are that the the configurations have been blanked because of the `handle_info/3` match _in this module_ that peeks into the backend state and blanks the configurations.

I do wonder whether it would be better not to blank the configurations at all, removing [that clause](https://github.com/nerves-networking/vintage_net_wizard/blob/338877f79b96e9ba1c2482260b95a1a44d7c66d8/lib/vintage_net_wizard/backend_server.ex#L312-L316) altogether.

Reply:
@mattludwigs - I'm making a pass on the WiFi wizard to fix another issue. I'm not familiar enough with this area to have opinions on this, but would it be possible to resolve this or decide that the patch is good enough so that it can be included in the next release?